src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends ColorSupplier {
+    private String color;
+    private int number;
+
+    public Ball() {
+        this.number = new Random().nextInt(101);
```

Comment:
`Random` object should be a field. Or you can use static method `Math.random()` without declaring any new object.